tests/fixtures_mc_simulator.py

```diff
@@ -17,15 +18,16 @@ def fixture_mantarray_mc_simulator():
         output_queue,
         error_queue,
         testing_queue,
-        read_timeout_seconds=SECONDS_TO_WAIT_WHEN_POLLING_QUEUES,
+        read_timeout_seconds=QUEUE_CHECK_TIMEOUT_SECONDS,
     )
 
     yield input_queue, output_queue, error_queue, testing_queue, simulator
 
 
 class MantarrayMCSimulatorNoBeacons(MantarrayMCSimulator):
-    def _handle_status_beacon(self) -> None:
-        return
+    def _send_status_beacon(self, truncate=False) -> None:
+        self._time_of_last_status_beacon_secs = time.perf_counter()
```

Comment:
thought for future---if `perf_counter` is imported from the `mc_simulator.py` module instead of `time`, then mocking it might have more of the expected effect.....maybe...I'm still not 100% sure exactly how namespaces work with mocking